src/Controller/ActualityController.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Article;
+use App\Entity\Image;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use App\Repository\ArticleRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Mapping\Entity;
+use Symfony\Component\HttpFoundation\Request;
+use App\Form\ArticleFormType;
+
+class ActualityController extends AbstractController
+{
+    /**
+     * @Route("/actualites", name="actuality")
+     */
+    public function showArticles(ArticleRepository $articleRepository): Response
+    {
+        $articles = $articleRepository->findBy(
+            ["section" => 'Actualites'],
+            ['id' => 'DESC'],
```

Comment:
You should not consider ids to sort your data by creation date. You should have a creation date field

Reply:
Ok, modified.